README.md

```diff
@@ -50,7 +50,7 @@ $default = function (RequestInterface $request) {
     return new Response();
 };
 
-$stack = new Stack($middleware, $default);
+$stack = new Stack($default, ...$middleware);
```

Comment:
As I said in #4, I am not at all convinced that using splat is beneficial here. This PR is more (to me) about moving `$default` to a property of the constructor, rather than using it as an argument for `dispatch()`.

What I'd like to see is:

- Rename this PR to make it clear that `$default` is moving
- Remove the usage of splat operator